internal/slice/slice.go

```diff
@@ -0,0 +1,16 @@
+package slice
+
+const (
+	Wildcard    = "*"
+	X509Subject = "x509.subject"
+)
+
+// Contains is a utility function to check if a string exists in an array
+func Contains(val string, values []string) bool {
```

Comment:
Should we rename it to StringContains or ContainsString ?

Reply:
Yeah, ContainsString might be better.